#! /bin/sh /usr/share/dpatch/dpatch-run
## 30_top.1_cpustate.dpatch by  <csmall@localhost.localdomain>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: top CPU state descriptions

@DPATCH@
diff -urNad --exclude=CVS --exclude=.svn ./top.1 /tmp/dpep-work.9EV6k0/procps-3.2.6/top.1
--- ./top.1	2005-10-30 13:37:43.000000000 +1100
+++ /tmp/dpep-work.9EV6k0/procps-3.2.6/top.1	2005-10-31 09:24:14.000000000 +1100
@@ -533,6 +533,37 @@
 upper case\fR letter and to the\fB right\fR with the\fB lower case\fR
 letter.
 
+.\" ......................................................................
+.SS 2c. CPU States
+.\" ----------------------------------------------------------------------
+The CPU states are shown in the Summary Area. They are always shown as a 
+percentage and are for the time between now and the last refresh.
+
+.TP 3
+\fB us\fR \*(EM User CPU time
+The time the CPU has spent running users' processes that are not
+niced.
+
+.TP 3
+\fB sy\fR \*(EM System CPU time
+The time the CPU has spent running the kernel and its processes.
+
+.TP 3
+\fB ni\fR \*(EM Nice CPU time
+The time the CPU has spent running users' proccess that have been
+niced.
+
+.TP 3
+\fB wa\fR \*(EM iowait
+Amount of time the CPU has been waiting for I/O to complete.
+
+.TP 3
+\fB hi\fR \*(EM Hardware IRQ
+The amount of time the CPU has been servicing hardware interrupts.
+
+.TP 3
+\fB si\fR \*(EM Software Interrupts
+The amount of time the CPU has been servicing software interrupts.
 
 .\" ----------------------------------------------------------------------
 .SH 3. INTERACTIVE Commands
